uiowa/uiowa
# Limit number of slides to 4 for slider

Resolves #4645 and #4857

# How to test

4645:
1. `blt ds --site=default`
2. `drush @default.local cim`
3. Add a slider. You should not be able to add more than 4 slides. 
4. Is the help text helpful?
5. Does it work fine with 4 slides?

4857:
1. `blt ds --site=default`
2. `blt frontend`
3. Add an edge-to-edge section.
4. Add a slider with one slide with a lot of slide headline text.
5. Does the text fit within the slide?
